Makefile

```diff
@@ -155,6 +160,27 @@ test-whonix: test-prereqs ## Runs tests for SD Whonix VM
 test-gpg: test-prereqs ## Runs tests for SD GPG functionality
 	python3 -m unittest discover -v tests -p test_gpg.py
 
+PHONY: run-client
+run-client: assert-dom0 ## Run client application (automatic login)
+ifeq ($(XDOTOOL_PATH),)
+	@echo $(XDOTOOL_PATH)
+	@echo 'please install xdotool with "sudo qubes-dom0-update xdotool"'
+	@false
+endif
+ifeq ($(OATHTOOL_PATH),)
+	@echo 'please install oathtool with "sudo qubes-dom0-update oathtool"'
+	@false
+endif
+	sdw-updater
```

Comment:
Maybe `qvm-run sd-app securedrop-client` so it doesn't accidentally trigger the updater and type into it?